var/spack/repos/builtin/packages/hypar/package.py

```diff
@@ -47,17 +47,17 @@ class Hypar(AutotoolsPackage):
     def configure_args(self):
         args = []
         spec = self.spec
-        if "+mpi" in spec:
+        if spec.satisfies("+mpi"):
             args.append("--enable-mpi")
```

Comment:
`--enable-mpi` is not necessary; adding that argument doesn't do anything.

Reply:
Good to know. This will be best addressed in a separate PR to keep this PR on scope.

Reply:
Thanks; I have addressed this in a [new PR](https://github.com/spack/spack/pull/46394). I also needed to update the HyPar's repo and url since it moved to github.